File: Programacion/Ejercicios_Clase/ejercicio09/Main.java
public class Main {
    public static void main(String[] args) {
        // Para cada tipo de bucle vamos a
        // imprimir desde el -2 al 7
        loopFor();
        loopWhile();
        loopDoWhile();
    }

    public static void loopFor() {
        int min = -2;
        int max = 7;
        System.out.println("Bucle for de " + min + " a "+ max);
        for(int i = min; i <= max; i++) {
            System.out.println(i+"");
        }
    }
    
    public static void loopWhile() {
        int min = -2;
        int max = 7;
        System.out.println("Bucle while de " + min + " a "+ max);
        
        int i = min;
        while (i >= min && i <= max) {
            System.out.println(i+"");
            i++;
        }
    }
    
    public static void loopDoWhile() {
        int min = -2;
        int max = 7;
        System.out.println("Bucle do-while de " + min + " a "+ max);
        
        int i = min;
        do {
            System.out.println(i+"");
            i++;
        } while (i >= min && i <= max);
    }
}

File: Programacion/Ejercicios_Clase/ejercicio09/main.js

console.log("Hola desde java script")

const minimo = -2;
const maximo = 7;

loopFor(minimo, maximo);
loopWhile(minimo, maximo);
loopDoWhile(minimo, maximo);

function loopFor(min, max) {
    console.log("Bucle for de " + min + " a "+ max);
    for(let i = min; i <= max; i++) {
        console.log(i+"");
    }
}

function loopWhile(min, max) {
    console.log("Bucle while de " + min + " a "+ max);
    
    let i = min;
    while (i >= min && i <= max) {
        console.log(i+"");
        i++;
    }
}

function loopDoWhile(min, max) {
    console.log("Bucle do-while de " + min + " a "+ max);
    
    let i = min;
    do {
        console.log(i+"");
        i++;
    } while (i >= min && i <= max);
}

console.log("adios")
